ShowMyData/Pages/Index.cshtml.cs

```diff
@@ -12,19 +12,25 @@ namespace ShowMyData.Pages
 {
     public class IndexModel : PageModel
     {
-        public void OnGet()
+        public JsonResult OnGet()
```

Comment:
Returned json output

Reply:
Careful... this is changing the functionality of the app.  It's currently returning an HTML page, and you're changing this to return JSON... the group might want to have an HTML page here.